### ISSUE TITLE
文本分类Demo因读入Dataset训练时未对batch做padding引起Blocking queue is killed because the data reader raises an exception

### ISSUE THREAD

**Issue by vForce825** [PaddlePaddle/PaddleFormers#1269]
- 版本、环境信息
1）PaddleHub2.0.1，paddlepaddle-gpu 2.0.0
2）Ubuntu 16.04 Python 3.7.0
- 最小复现代码
import paddle
import paddlehub as hub
model = hub.Module(name='roberta-wwm-ext-large', task='seq-cls', num_classes=2)
train_dataset = hub.datasets.ChnSentiCorp(
tokenizer=model.get_tokenizer(), max_seq_len=128, mode='train')
dev_dataset = hub.datasets.ChnSentiCorp(
tokenizer=model.get_tokenizer(), max_seq_len=128, mode='dev')
optimizer = paddle.optimizer.Adam(learning_rate=5e-5, parameters=model.parameters())
trainer = hub.Trainer(model, optimizer, checkpoint_dir='test_ernie_text_cls')
trainer.train(train_dataset, epochs=3, batch_size=32, eval_dataset=dev_dataset)

- 原因排查
观察发现报错为：
File "/home/daniel/anaconda3/lib/python3.7/site-packages/paddle/fluid/dataloader/dataloader_iter.py", line 90, in default_collate_fn
tmp = np.stack(slot, axis=0)
排查一个batch内的数据，发现每个样本的shape不相同，所以引起np.stack报错

**Comment by KPatr1ck** [comment]
你好，该问题目前正在修复中
如果需求紧急，可以先执行`pip install paddlenlp==2.0.0rc2`安装旧版本的paddlenlp依赖，再运行脚本。

**Comment by vForce825** [comment]
谢谢，2.0.0rc2可以正常运行。
但是不像以前paddlehub可以使用多卡，目前的trainer里并没有multi_gpu相关的参数，网上也找不到多卡训练相关的API说明。想问下现在多卡训练使用什么API启动呢？
